README.md

```diff
@@ -9,6 +9,8 @@ Csv file creation was added to [Gecko](https://github.com/judyfong/gecko/pull/1)
 
 It is assumed that the corpus data will exist within the data directory specified at the top of the [create_corpus](create_corpus.sh) bash script.
 
+This repository is listed under the other software section in the [Awesome Diarization repository](https://github.com/wq2012/awesome-diarization) that contains list of awesome Speaker Diarization papers, libraries, datasets and more.
+
```

Comment:
Remove “that contains” and change to “which is” please. If you want an explanation of why this change just let me know.

Reply:
In fact just shorten the sentence to about ten 10 words if you can please. Here are tips insugural seminal leading frontier

Reply:
Inaugural

first google result in the us etc

Reply:
I shortened it and hopefully made it more inaugural

Reply:
That's not what I meant but I like it so I'll leave it in. I meant write the following:
```suggestion
This repository is listed under the other software section in [Awesome Diarization, a GitHub repository](https://github.com/wq2012/awesome-diarization) which lists information about diarization.
```